lib/markbind/lib/parser.js

```diff
@@ -49,9 +49,10 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
-function calculateBoilerplateFilePath(filePath, config) {
-  const base = calculateNewBaseUrl(filePath, config.rootPath, config.baseUrlMap).relative;
-  return path.resolve(base, BOILERPLATE_DIRECTORY_NAME, path.basename(filePath));
+function calculateBoilerplateFilepath(filePath, pathWithinBoilerplate, config) {
```

Comment:
Why the change from `*FilePath` to `*Filepath`? The latter is inconsistent with the rest of this file.